### PR TITLE
New Data for Workshop

### PR DESCRIPTION
**Pull request by elbfox**
Updates to MOGA-generated f_nrge/f_norm (NormalizedMatrix), www/maps, Decisions, and small ui.R server.R and WSM.R changes.

### CODE REVIEW

**Comment by elbfox** [comment]
Oh I  just went through and resolved the conflicts. Looks like it didn't work! 

This PR has updates to www/maps, f_nrge/f_norm (they're not working locally...I'm trying all different things to troubleshoot here but wanted you to have the updated versions @sythel ), DamsData_Workshop.csv and Decisions_workshop.csv

**Comment by sythel** [comment]
f_raw  is sourced but missing from github

**Comment by sythel** [comment]
ready for review?

**Comment by elbfox** [comment]
Yes, I'm stuck where I tried to add f_raw but my branch is now behind because I resolved the conflicts here. I can addit with the next merge...f_raw was to help us check for errors in normalization and we actually don't need it in the script unless you call on it explicitly somewere that I haven't noticed

File: src/dams_mcda/ui.R [hunk 1]
```diff
@@ -74,18 +74,16 @@ ui <- shinyUI(fluidPage(
...
+	div(
+		id="progress-tracker",
+		htmlOutput("TotalProgress")
+	),
 
-	# bottom of page total progress tracker
-	div(id="progress-tracker", htmlOutput("TotalProgress")),
+	# intro popup-modal, forces user to decide input before proceeding
 
-	# navigation buttons
```
Comment (sythel) [inline src/dams_mcda/ui.R]: want to keep this here

File: src/dams_mcda/ui.R [hunk 2]
```diff
@@ -1864,7 +1862,9 @@ ui <- shinyUI(fluidPage(
 				 "
 			 )
 		)
-	) # end of navigation panel
+	), # end of navigation panel
```
Comment (sythel) [inline src/dams_mcda/ui.R]: remove these

File: src/dams_mcda/ui.R [hunk 3]
```diff
@@ -74,18 +74,16 @@ ui <- shinyUI(fluidPage(
...
+		id="progress-tracker",
+		htmlOutput("TotalProgress")
+	),
 
-	# bottom of page total progress tracker
-	div(id="progress-tracker", htmlOutput("TotalProgress")),
+	# intro popup-modal, forces user to decide input before proceeding
 
-	# navigation buttons
-	div(id="nav-buttons",
```
Comment (sythel) [inline src/dams_mcda/ui.R]: keep

**Comment by sythel** [comment]
i will fix

**Pull request merged**